recipe/activate.sh

```diff
@@ -1,10 +1,26 @@
 #!/bin/bash
-if [ "$(uname)" == "Darwin" ]
-then
-    # for Mac OSX
-    export KERAS_BACKEND=tensorflow
-elif [ "$(uname)" == "Linux" ]
-then
-    # for Linux
-    export KERAS_BACKEND=theano
+
+# get the location of the keras.json file
+KERAS_JSON="$HOME/.keras/keras.json"
+
+if [ -f $KERAS_JSON ]; then # if the file exists
+
+  BACKEND_KEY="backend" # value we are interested in finding
+  KEY_VALUE_REGEX="\"($BACKEND_KEY)\": \"([^\"]*)\"" # regex
+  JSON_TEXT=$(cat $KERAS_JSON) # data in keras.json file
+
+  if [[ $JSON_TEXT =~ $KEY_VALUE_REGEX ]]; then # check for a match
+    BACKEND_VALUE="${BASH_REMATCH[2]}" # get the user defined backend value
+    export KERAS_BACKEND=$BACKEND_VALUE # set the environmental variable
+  fi
```

Comment:
Yes, that way the user can specify which backend they want to use based on the `.keras/keras.json` file. If it's absolutely critical that the conda-forge Keras recipe sets the backend based on the system then at the very least there needs to be a warning during installation that the `keras_activation.sh` script will override the user defined variables in `keras.json`. 

This PR maintains the current functionality but will set the `KERAS_BACKEND` variable based on the user specified value. I had a lot of headaches trying to set the backend through the `keras.json` file based on the Keras documentation until someone pointed out that the conda recipe was overriding that on environment activation.